donut_shop.html

```diff
@@ -5,33 +5,25 @@
 </head>
 <body bgcolor="#FACC2Eg">
   <h1>DonutShop</h1>
-  <table style="width:100%">
+  <table id= "tbl" style="width:100%">
     <h1><center><font color="#FF0080"> Top Pot Donuts Works for You!</center></h1>
     <tr>
-    <tr>
        <th>Location</th>
-       <td>7am</td>
-       <td>8am</td>
-       <td>9am</td>
-       <td>10am</td>
-       <td>11am</td>
-       <td>12pm</td>
-       <td>1pm</td>
-       <td>2pm</td>
-       <td>3pm</td>
-       <td>4pm</td>
-       <td>5pm</td>
-       <td>6pm</td>
-       <td>7pm</td>
-    </tr>
-    <tr>
-    <tr id = "downtown"></tr>
-    <tr id = "capHill"></tr>
-    <tr id = "southlake"></tr>
-    <tr id = "wedgeWood"></tr>
-    <tr id = "ballard"></tr>
+       <th>7am</th>
+       <th>8am</th>
+       <th>9am</th>
+       <th>10am</th>
+       <th>11am</th>
+       <th>12pm</th>
+       <th>1pm</th>
+       <th>2pm</th>
+       <th>3pm</th>
+       <th>4pm</th>
+       <th>5pm</th>
+       <th>6pm</th>
+       <th>7pm</th>
+       <th>Total</th>
     </tr>
-
   </table>
 
```

Comment:
You can remove these extra line breaks in your HTML. Be sure to compress it down into a tight readable format.